themes/bootstrap3/templates/combined/results-list.phtml

```diff
@@ -28,15 +28,28 @@
       <? foreach (($top = $results->getRecommendations('top')) as $current): ?>
         <?=$this->recommend($current)?>
       <? endforeach; ?>
-      <?=$this->transEsc("Showing")?>
-      <strong><?=$this->localizedNumber($results->getStartRecord())?></strong> - <strong><?=$this->localizedNumber($results->getEndRecord())?></strong>
-      <? if (!isset($view->skipTotalCount)): ?>
-        <?=$this->transEsc('of')?> <strong><?=$this->localizedNumber($recordTotal)?></strong>
+      <?
+        $transParams = [
```

Comment:
Something is wrong here -- when I test combined search, I only see query times on all of the columns -- the range text is missing.